mne_bids/utils.py

```diff
@@ -267,7 +270,7 @@ def make_dataset_description(path, name=None, data_license=None,
     pop_keys = [key for key, val in description.items() if val is None]
     for key in pop_keys:
         description.pop(key)
-    _write_json(description, fname, verbose=verbose)
+    _write_json(description, fname, overwrite=True, verbose=verbose)
```

Comment:
I am not particularly happy with this, but it is actually the same as the current behavior.